src/transaction.ts

```diff
@@ -402,15 +402,38 @@ export class Snapshot extends EventEmitter {
    * @see [ReadRequest API Documentation](https://cloud.google.com/spanner/docs/reference/rpc/google.spanner.v1#google.spanner.v1.ReadRequest)
    *
    * @typedef {object} ReadRequest
-   * @property {string[]|string[][]} [keys] The primary keys of the rows in this table to be
+   * @property {string} table The name of the table in the database to be read.
+   * @property {string[]} columns The columns of the table to be returned for each
+   *     row matching this query.
+   * @property {string[]|string[][]} keys The primary keys of the rows in this table to be
    *     yielded. If using a composite key, provide an array within this array.
    *     See the example below.
    * @property {KeyRange[]} [ranges] An alternative to the keys property; this can
-   *       be used to define a range of keys to be yielded.
+   *     be used to define a range of keys to be yielded.
+   * @property {string} [index] The name of an index on the table.
    * @property {boolean} [json=false] Receive the rows as serialized objects. This
    *     is the equivalent of calling `toJSON()` on each row.
-   * @property {JSONOptions} [jsonOptions] Configuration options for the
-   *     serialized objects.
+   * @property {JSONOptions} [jsonOptions] Configuration options for the serialized
+   *     objects.
+   * @property {object} [keySet] Defines a collection of keys and/or key ranges to
+   *     read.
+   * @property {number} [limit] The number of rows to yield.
+   * @property {Buffer} [resumeToken]
```

Comment:
This is normally not something a user needs to specify, as resuming a broken stream is something that is automatically handled internally in the `PartialResultSetStream`. I would therefore suggest either leaving this out of the documentation, or otherwise indicate that this is something users normally do not need to use themselves.

Reply:
Removed from the documentation!.